index.html

```diff
@@ -2,45 +2,199 @@
+        </div>
+    </div>
+    <br>
+
+    <!-- Buttons for sharing, copying and follwing play link -->
+    <div>
+        <button class="mdc-icon-button material-icons" id="randomWord" onclick="pickRandomWord()"
+            style="margin-right: 2em;">
+            <div class="mdc-icon-button__ripple"></div>
+            shuffle
+        </button>
+        <button class="mdc-icon-button material-icons" onclick="copyLink()">
+            <div class="mdc-icon-button__ripple"></div>
+            content_copy
+        </button>
+        <button class="mdc-icon-button material-icons" onclick="shareLink()">
+            <div class="mdc-icon-button__ripple"></div>
+            share
+        </button>
+        <button class="mdc-icon-button material-icons" onclick="playWord()">
+            <div class="mdc-icon-button__ripple"></div>
+            play_arrow
         </button>
     </div>
 
-    <div style="margin-bottom: 1em;">
-        <select id="language" onchange="updateLanguage(this.value)">
-            <option value="eng"> ENG </option>
-            <option value="slo"> SLO </option>
-        </select>
-        <input type="text" id='inputText' autofocus placeholder="Enter word" autofocus
-            onkeyup="updateNovaBeseda(this.value)" maxlength="5">
 
-    </div>
 
 
 
-    <div style="margin-bottom: 1em;">
-        Hide the word: <input type="checkbox" id="myCheck" onclick="hide()">
-    </div>
+    <script>
 
+        // Activate JS for text and button components
+        const choice = mdc.textField.MDCTextField.attachTo(document.querySelector('.mdc-text-field'))
+        mdc.textField.MDCTextFieldCharacterCounter.attachTo(document.querySelector('.mdc-text-field-character-counter'))
+        for (const btn of document.querySelectorAll('.mdc-button')) {
+            mdc.ripple.MDCRipple.attachTo(btn)
+        }
 
+        // Dynamic elements
+        const outline = document.getElementById("outline")
+        const visibility = document.getElementById("visibility")
+        const error = document.getElementById("error")
+        const language = document.getElementById("language")
 
 
-    <button id="randomWord" onclick="pickRandomWord()">
-        Pick a random word
-    </button>
-    <script>
+
+        function toggleVisibility() {
+            if (isTextVisible()) {
+
+                choice.input.setAttribute('type', 'password')
+                visibility.innerHTML = 'visibility_off'
+            } else {
+                choice.input.setAttribute('type', 'text')
+                visibility.innerHTML = 'visibility'
+            }
+        }
+
+        function isTextVisible() {
+            return choice.input.getAttribute('type') == 'text'
+        }
+
+
+        function toggleLanguage() {
+            if (jezik === 'eng') {
+                jezik = 'slo'
+            } else {
+                jezik = 'eng'
+            }
+            language.setAttribute('src', `${jezik}.png`)
```

Comment:
You forgot to commit images `slo.png` and `eng.png` so these now appear as broken images.